### PR TITLE
Nwchem added cyclecloud support

### PR DESCRIPTION
**Pull request by garvct**
Added cyclecloud support to nwchem.

### CODE REVIEW

File: apps/nwchem/readme.md [hunk 1]
```diff
@@ -36,10 +36,44 @@ azhpc-connect -u hpcuser headnode
 ## Run the NWCHEM h2o_freq scenario
 To run on a single node with 4 cores run
 ```
-qsub -l select=1:ncpus=4:mpiprocs=4 $azhpc_dir/apps/nwchem/run_h2o_freq.sh
+qsub -l select=1:ncpus=60:mpiprocs=4 $HOME/apps/nwchem/run_h2o_freq.sh
 ```
 
 To run on two HB nodes with 8 total cores (4 cores on each node) run
 ```
 qsub -l select=2:ncpus=60:mpiprocs=4 $azhpc_dir/apps/nwchem/run_h2o_freq.sh
```
Comment (kanchanm) [inline apps/nwchem/readme.md]: just apps/nwchem/run_h2o_freq.sh to be consistent

File: apps/nwchem/readme.md [hunk 2]
```diff
@@ -36,10 +36,44 @@ azhpc-connect -u hpcuser headnode
 ## Run the NWCHEM h2o_freq scenario
 To run on a single node with 4 cores run
 ```
-qsub -l select=1:ncpus=4:mpiprocs=4 $azhpc_dir/apps/nwchem/run_h2o_freq.sh
+qsub -l select=1:ncpus=60:mpiprocs=4 $HOME/apps/nwchem/run_h2o_freq.sh
```
Comment (kanchanm) [inline apps/nwchem/readme.md]: apps/nwchem/run_h2o_freq.sh

File: apps/nwchem/readme.md (hunk 1 above)
Comment (garvct) [reply apps/nwchem/readme.md]: Done.

File: apps/nwchem/readme.md (hunk 2 above)
Comment (garvct) [reply apps/nwchem/readme.md]: done

**Pull request merged**